lib/Serialization/ModuleFile.cpp

```diff
@@ -23,7 +23,7 @@
 #include "swift/ClangImporter/ClangImporter.h"
 #include "swift/Serialization/BCReadingExtras.h"
 #include "swift/Serialization/SerializedModuleLoader.h"
-
+#include "clang/Basic/Module.h"
```

Comment:
Ick. Can we indirect this through ClangModuleUnit somehow?

Reply:
I guess so. Maybe we'll end up using this again somewhere.